Sources/protoc-gen-swift/GeneratorOptions.swift

```diff
@@ -49,6 +49,7 @@ class GeneratorOptions {
   let outputNaming: OutputNaming
   let protoToModuleMappings: ProtoFileToModuleMappings
   let visibility: Visibility
+  let extraModuleImports: [String]?
```

Comment:
If I'm reading the code correctly, this is always getting set, so the optional part doesn't seem worth while.  Does the generation code then also not need to be an `if let` but instead a check to see if it is empty?